sonar-plugin-api/src/main/java/org/sonar/api/ce/posttask/PostProjectAnalysisTask.java

```diff
@@ -71,11 +72,23 @@
      * Date of the analysis.
      * <p>
      * This date is the same as the date of the project analysis report and the snapshot.
-     * 
+     *
+     * @deprecated use {@link #getAnalysisDate()} instead. When {@link #getAnalysisDate()} returns
+     *             {@link Optional#empty() empty}, the current date will be returned.
      */
+    @Deprecated
     Date getDate();
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)